### PR TITLE
Update all documents

### PR DESCRIPTION
**Pull request by sioncojp**
### Overview
- The notation is based on terraform-provider-aws.
- I've copied the comment out listed at the following URL .
    - https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go
    - https://github.com/googleapis/google-api-go-client/blob/master/groupssettings/v1/groupssettings-gen.go
- The `data source` is described simply because it is a value to be obtained

### issue
#88

### CODE REVIEW

File: website/docs/d/group.md [hunk 1]
```diff
@@ -32,18 +32,18 @@ The following arguments are supported:
...
+* `description` - Description of the group.
 
-* `direct_members_count`
+* `direct_members_count` - Group direct members count.
 
-* `admin_created`
+* `admin_created` - Is the group created by admin.
 
-* `non_editable_aliases`
+* `non_editable_aliases` - List of non editable aliases.
```
Comment (sioncojp) [inline website/docs/d/group.md:47]: The above was copied from the URL
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2411

File: website/docs/d/group_settings.md [hunk 2]
```diff
@@ -36,116 +36,149 @@ The following arguments are supported:
...
 
-* `who_can_unmark_favorite_reply_on_any_topic`
+* `who_can_unmark_favorite_reply_on_any_topic` - Deprecated. This is merged into
+  the new whoCanAssistContent setting.
 
-* `who_can_view_group`
+* `who_can_view_group` - Permissions to view group messages.
 
-* `who_can_view_membership`
+* `who_can_view_membership` - Permissions to view membership.
```
Comment (sioncojp) [inline website/docs/d/group_settings.md:184]: The above was copied from the URL
https://github.com/googleapis/google-api-go-client/blob/master/groupssettings/v1/groupssettings-gen.go#L146

File: website/docs/d/user.md [hunk 3]
```diff
@@ -28,63 +28,68 @@ The following arguments are supported:
...
+* `custom_schema` - Custom fields of the user.
 
-* `external_ids` - contains a list of sets containing `custom_type`, `type` and
-  `value`.
+* `external_ids` - A list of external IDs for the user, such as an employee or network ID. 
+  contains a list of sets containing `custom_type`, `type` and `value`.
 
-* `organizations` - contains a list of sets containing `cost_center`,
+* `organizations` - A list of organizations the user belongs to.
+  contains a list of sets containing `cost_center`,
```
Comment (sioncojp) [inline website/docs/d/user.md:93]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L3665

File: website/docs/r/domain.md [hunk 4]
```diff
@@ -31,9 +31,9 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `creation_time`
+* `creation_time` - Creation time of the domain.
 
-* `etag`
+* `etag` - ETag of the resource.
```
Comment (sioncojp) [inline website/docs/r/domain.md:36]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2184

File: website/docs/r/user_schema.md [hunk 5]
```diff
@@ -252,7 +252,7 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `schema_id`
+* `schema_id` - Unique identifier of Schema.
```
Comment (sioncojp) [inline website/docs/r/user_schema.md:255]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L3320

File: website/docs/r/user.md [hunk 6]
```diff
@@ -125,53 +125,57 @@ The following arguments are supported:
...
+* `2s_enforced` - Is 2-step verification enforced.
 
-* `2s_enrolled`
+* `2s_enrolled` - Is enrolled in 2-step verification.
 
-* `is_mailbox_setup`
+* `is_mailbox_setup` - Is mailbox setup.
 
-* `last_login_time`
+* `last_login_time` - User's last login time.
```
Comment (sioncojp) [inline website/docs/r/user.md:178]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L3665

File: website/docs/r/group_member.md [hunk 7]
```diff
@@ -41,13 +41,13 @@ The following arguments are supported:
...
+* `etag` - ETag of the resource.
 
-* `kind`
+* `kind` - Kind of resource this is.
 
-* `status`
+* `status` - Status of member.
 
-* `type`
+* `type`- Type of member.
```
Comment (sioncojp) [inline website/docs/r/group_member.md:50]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2513

File: website/docs/r/group_members.md [hunk 8]
```diff
@@ -49,12 +49,12 @@ The following arguments are supported:
...
-  * `kind`
-  * `status`
-  * `type`
-  * `role`
+  * `email` - Email of member.
+  * `etag` - ETag of the resource.
+  * `kind` - Kind of resource this is.
+  * `status` - Status of member.
+  * `type` - Type of member.
+  * `role` - Role of member.
```
Comment (sioncojp) [inline website/docs/r/group_members.md:57]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2568

File: website/docs/r/group.md [hunk 9]
```diff
@@ -29,19 +29,19 @@ The following arguments are supported:
...
 In addition to the above arguments, the following attributes are exported:
 
-* `direct_members_count`
+* `direct_members_count` - Group direct members count.
 
-* `admin_created`
+* `admin_created` - Is the group created by admin.
 
-* `non_editable_aliases`
+* `non_editable_aliases` - List of non editable aliases.
```
Comment (sioncojp) [inline website/docs/r/group.md:44]: The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2411

File: website/docs/r/user.md [hunk 10]
```diff
@@ -88,23 +88,25 @@ The following arguments are supported:
   email addresses.
 
 * `include_in_global_list` - (Optional) Boolean switch to show or hide this user
-  in the global list. Defaults to true.
+  in the global list. 
+  Valid values are `true` or `false`. Defaults to `true`.
 
 * `is_ip_whitelisted` - (Optional) Boolean switch to enforce whitelisting of the
   user's IP.
+  Valid values are `true` or `false`. Defaults to `false`.
```
Comment (sioncojp) [inline website/docs/r/user.md:96]: from : https://github.com/DeviaVir/terraform-provider-gsuite/blob/master/gsuite/resource_user.go#L119

**Review by DeviaVir — APPROVED** [review]
:+1: this is a great addition, thank you!

**Pull request merged**